devel/sbt/Portfile

```
@@ -20,17 +20,17 @@ long_description \
     and testing with triggered execution in mixed Scala/Java projects.
 
 homepage        https://www.scala-sbt.org/
-master_sites    https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/${version}
+master_sites    https://repo1.maven.org/maven2/org/scala-sbt/sbt-launch/${version}
 distname        ${name}-launch
 dist_subdir     ${name}/${version}
 
-checksums       md5     c660658ea9b1300d31aaa713f214b1b4 \
-                sha1    db44e7b60ed5c82ea00315ad223e3a8cc3d086e5 \
-                rmd160  a61cd251a1593dafa0d1452dbb615b4fb1fd5a19 \
-                sha256  add170a6e07d3ecf2a5add54565a0b5f729524083f0b05f9e1d4df90adb9257b \
-                size    1210278
+checksums       md5     0ccfcf9786148578779fc3a0c48c7b3c \
+                sha1    6bb9ba1cd6e57b3d7fc62db961af11fd2de83633 \
+                rmd160  2b37784d9185d3722b4a18cbe722bf2148ca0fa3 \
```

There's probably no reason to list md5 or sha1 checksums anymore.
```suggestion
checksums       rmd160  2b37784d9185d3722b4a18cbe722bf2148ca0fa3 \
```